### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DyerSituation**
added `conflict`, changed some styling

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,22 +166,73 @@ For more information on test writing, please visit our [documentation](http://su
...
+<pre>--tags <b>tag</b>[<b>,tag,...</b>]</pre> 
 
+###Site-ID
+ only run tests for a specific site. Get in touch with us for help on getting that you site id if you are unable to.
+
+<pre>--site-id <b>ID</b></pre>
+
+###Folder-ID
+ run tests in specified folder.
+<pre>--folder ID <b>ID</b></pre>
```
Comment (epaulet) [inline README.md]: `ID` appears twice

File: README.md [hunk 2]
```diff
@@ -166,22 +166,73 @@ For more information on test writing, please visit our [documentation](http://su
...
+###Crowd
+<pre>--crowd [<b>default</b>|<b>on_premise_crowd</b>]</pre>
+
+
+###Conflict
+use the abort option to abort any runs in progress in the same environment as your new run. use the abort-all option to abort all runs in progress.
+- <pre>--conflict <b>option</b></pre> </pre>
+
+
+###Fg
```
Comment (epaulet) [inline README.md]: This is kind of a weird header. Why not just call it what it is? `Foreground`

Comment (ukd1) [inline README.md]: @DyerSituation can you fix this?

**Comment by ukd1** [comment]
@DyerSituation can you make this mergeable so we can ship plz?

File: README.md (hunk 2 above)
Comment (DyerSituation) [inline README.md]: I thought it would be better to use the command names as headers. Is it better to just use Foreground?

**Comment by ukd1** [comment]
@DyerSituation can you finish this before you go plz?

File: README.md (hunk 2 above)
Comment (epaulet) [inline README.md]: If it's a header it should probably describe the behavior.

File: README.md [hunk 3]
```diff
@@ -214,6 +214,65 @@ api to construct a junit report.  This is useful to track tests in CI such as Je
 - `--import-variable-name NAME` - Use with `run` and `import-variable-csv-file` to upload
 new tabular variable values before your run to specify the name of your tabular variable.
 
+###Site-ID
+ only run tests for a specific site. Get in touch with us for help on getting that you site id if you are unable to.
+
+<pre>--site-id <b>ID</b></pre>
+
+###Folder-ID
+ run tests in specified folder.
```
Comment (ukd1) [inline README.md]: extra space at the start of this line

File: README.md [hunk 4]
```diff
@@ -214,6 +214,65 @@ api to construct a junit report.  This is useful to track tests in CI such as Je
 - `--import-variable-name NAME` - Use with `run` and `import-variable-csv-file` to upload
 new tabular variable values before your run to specify the name of your tabular variable.
 
+###Site-ID
+ only run tests for a specific site. Get in touch with us for help on getting that you site id if you are unable to.
```
Comment (ukd1) [inline README.md]: extra space at the start of this line

File: README.md [hunk 5]
```diff
@@ -214,6 +214,65 @@ api to construct a junit report.  This is useful to track tests in CI such as Je
...
+<pre>--crowd [<b>default</b>|<b>on_premise_crowd</b>]</pre>
+
+
+###Conflict
+use the abort option to abort any runs in progress in the same environment as your new run. use the abort-all option to abort all runs in progress.
+- <pre>--conflict <b>option</b></pre> </pre>
+
+
+###Foreground
+results in the foreground - rainforest-cli will not return until the run is complete. This is what you want to make the build pass / fail dependent on rainforest results
```
Comment (ukd1) [inline README.md]: > results in the foreground - rainforest-cli will not return until the run is complete. This is what you want to make the build pass / fail dependent on rainforest results

-->

> Show live progress in the foreground - rainforest-cli will not return until the run is complete. The exit code will correspond to the build status - which is perfect for using in a continuous integration server.

File: README.md [hunk 6]
```diff
@@ -214,6 +214,65 @@ api to construct a junit report.  This is useful to track tests in CI such as Je
...
+- <pre>--fg</pre>
+
+
+###Wait
+wait for an existing run to finish instead of starting a new one, and exit with a non-0 code if the run fails. rainforest-cli will exit immediately if the run is already complete.
+- <pre>--wait <b>RUN_ID</b></pre>
+
+###Fail-fast
+fail the build as soon as the first failed result comes in. If you don't pass this it will wait until 100% of the run is done. Use with --fg.
+
```
Comment (ukd1) [inline README.md]: why so much spacing?

**Comment by ukd1** [comment]
@DyerSituation this looks great, but some really minor grammar / formatting things to fix

**Pull request merged**